### DISCUSSION TITLE
Possible Typo in param 'energy_kWhdoy'

### DISCUSSION THREAD

**Discussion by simeonovp** [community/community#3119] [Other Feature Feedback, Questions, & Ideas] [answered]
Is this a know issue?

**Comment by evi-liu** [comment]
Hi there! Could you please provide more information or screenshots on where you're seeing this?

**Comment by simeonovp** [comment]
Hi @evi-liu.  Steps to reproduce:
1. run command 'decode-config.py -d <ipOrHost> -o <filepath> --json-indent, 2'
2. open resulting file <filepath> in text editor and search for "energy_kWhdoy" (in my file is line 132)

FYI my environment is:
    "env": {
      "platform": "Linux-4.19.75-v7l+-armv7l-with-debian-10.2", 
      "python": "3.7.3", 
      "script": "decode-config.py v9.2.0.2 [3141b7e]", 
      "system": "Linux armv7l 4.19.75-v7l+ #1270 SMP Tue Sep 24 18:51:41 BST 2019"
    }, 

PS. I would also recommend replacing '"weight_max": 0.0,' with '"weight_max": 0,'. When loading and saving the file with many tools, exactly that happens.

**Comment by simeonovp** [comment]
@evi-liu 
Additiona info: in the py-file I founf follow line:
    'energy_kWhdoy':                (Platform.ALL,   '<H',  0x378,       (None, None,                           ('Power',       None)) ),
;)

**Comment by evi-liu** [comment, marked as answer]
This forum is currently being used for only GitHub Discussions feedback. I'd recommend you redirect this question to a more specific forum.
